charts/kong/README.md

```diff
@@ -536,6 +537,7 @@ nodes.
 | SVC.ingress.path                   | Ingress path.                                                                         | `/`                 |
 | SVC.ingress.annotations            | Ingress annotations. See documentation for your ingress controller for details        | `{}`                |
 | SVC.annotations                    | Service annotations                                                                   | `{}`                |
+| SVC.annotations                    | Service labels                                                                        | `{}`                |
```

Comment:
The descriptions "service annotations" and "service labels" aren't very helpful because they essentially repeat the value name, and because the "service"/"SVC" term is overloaded here. Please say what effects seetting something there has.

An example of what I mean: "Annotations of the Kubernetes `Service` created for `SVC`"

Reply:
We explain the use of SVC above: https://github.com/Kong/charts/blame/kong-1.14.5/charts/kong/README.md#L482-L496

The description is brief, but that's true of most of these--a lot of them are just standard K8S Service features that don't need additional explanation specific to our chart. Leaving these as-is.